galcheat/utilities.py

```diff
@@ -0,0 +1,77 @@
+import astropy.units as u
+
+from galcheat.helpers import get_filter, get_survey
+
+
+def mag2counts(magnitude, survey_name, filter_name):
+    """Convert source magnitude to counts for a given filter of a survey
```

Comment:
We may mention that it consists of electron counts ?

Reply:
But is it really ? 
For me, this is photon count. To be electron counts it needs to be multiplied by the gain.
It does not change a thing for LSST because the gain is set to 1.0 but for other surveys this is relevant.

This comes back to another related question about the units of the gain currently in electron/adu that should be changed to electron/count if we would like them to be compatible.

Reply:
Hm I think that the conversion from photons to electrons depends on the quantum efficiency while the gain is transforming electrons to ADU (so it is in e-/ADU and going from e- to ADUs is done by dividing by the gain).

Reply:
Coming back to this, as we are using zeropoints indicating magnitudes giving 1e-/s, I would say that the fluxes we compute with those zeropoints should be in e-.

Reply:
See #93